mfhdf/libsrc/mfsd.c

```diff
@@ -4288,7 +4288,7 @@ intn
 SDgetcompinfo(int32         sdsid,     /* IN: dataset ID */
               comp_coder_t *comp_type, /* OUT: the type of compression */
               comp_info    *c_info)       /* OUT: ptr to compression information
-                    structure for storing the retrieved info */
+              structure for storing the retrieved info */
```

Comment:
I didn't touch this file. Must be a clang-format hiccup.

Reply:
Maybe cause it is outside the parens. I bet doxygen style would be better.